.github/workflows/push.yml

```diff
@@ -6,10 +6,7 @@ jobs:
     steps:
     - uses: actions/checkout@master
     - name: build-and-test
-      uses: agemooij/github-actions-sbt@master
```

Comment:
will this work without the docker image specified here? Or should we use another?

Reply:
I run the script locally it did work.

Reply:
`agemooij/github-actions-sbt` contains a `Dockerfile` that provides the environment where this action will run (see https://github.com/agemooij/github-actions-sbt/blob/master/Dockerfile)
I would expect that the action won't know how to do `sbt` without that image.

Reply:
Ok I will setup sbt in there.

Reply:
@maasg done.